packages/block-library/src/rss/index.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Server-side rendering of the `core/rss` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/rss` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content with received rss items.
+ */
+function render_block_core_rss( $attributes ) {
+	$rss = fetch_feed( $attributes['feedURL'] );
+
+	if ( is_wp_error( $rss ) ) {
+		return '<div class="components-placeholder"><div class="notice notice-error"><strong>' . __( 'RSS Error:' ) . '</strong> ' . $rss->get_error_message() . '</div></div>';
+	}
+
+	if ( ! $rss->get_item_quantity() ) {
+		$rss->__destruct();
+		unset( $rss );
```

Comment:
This lines were added because of memory leaks in older PHP versions. It was fixed in PHP 5.3: http://simplepie.org/wiki/faq/i_m_getting_memory_leaks

WordPress is increasing its minimum in WP 5.2 to PHP 5.6. Maybe we should do the same, as all new code is for WP 5.2.

I removed the unnecessary code.

Reply:
I opened a ticket for bumping the PHP minimum: https://github.com/WordPress/gutenberg/issues/13478

Reply:
Yes sure, this are only 4 extra lines. I added them back with a comment about PHP 5.2 compatibility

Reply:
Thanks, it looks like this PR is very close to land.